### DISCUSSION TITLE
How to configure and run TLS (example) if i have .crt file ?

### DISCUSSION THREAD

**Discussion by Mgalazka2323** [davidepianca98/KMQTT#52] [Q&A] [answered]
hello,
Can someone help me with .crt file and configuration ?
I tried with assets folder and my file copied in other place and send path to TLSConfig but without success :( 
Can I send string certificate to TLSConfig ?
Is any working example how to run it ? I need it for Android and IOS Platform :)
Best regards
 Mateusz

**Comment by davidepianca98** [comment, marked as answer]
Hello, you can check out this code snippet here https://github.com/davidepianca98/KMQTT/issues/48#issuecomment-2071509627 on how to get the file path from the Android assets. I don't currently have an example for iOS.

**Reply by Mgalazka2323** [reply]
Thank you for your support and great job !!
Mateusz
